ark/analysis/spatial_analysis_test.py

```diff
@@ -27,7 +27,7 @@ def _make_distance_matrix(enrichment_type, dist_lim):
                                 coords=[np.arange(rand_mat.shape[0]) + 1,
                                         np.arange(rand_mat.shape[1]) + 1])
 
-        fovs = ["Point8", "Point9"]
+        fovs = ["Fov8", "Fov9"]
```

Comment:
Let's make these lowercase, Point was only capitalized as an artifact of the Matlab code. 
I'm ambivalent about fov_1 or fov1, up to you which you think looks better

Reply:
Looks like most people have already used `fov1` formatting, so that's what I'll use.